cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/TopicController.java

```diff
@@ -200,6 +213,11 @@ public static TopicController fromConfigMap(ConfigMap kafkaClusterCm) {
                 // TODO : add parsing and validation
                 topicController.setZookeeperSessionTimeoutMs(zookeeperSessionTimeoutMs);
             }
+
+            Integer topicMetadataMaxAttempts = json.getInteger(TopicController.TOPIC_METADATA_MAX_ATTEMPTS_FIELD);
```

Comment:
I think we should switch to using the `json.get*()` method which take a default value. It would make this code much cleaner without all the `if` blocks, don't you think?